Add Entry SQL: Contains (#665)
Codecademy/docs: content/sql/concepts/commands/terms/contains/contains.md

---

> Title: 'CONTAINS'

Reviewer: After taking a closer look, let's actually move this to a term under ["operators"](https://www.codecademy.com/resources/docs/sql/operators) rather than commands. Thanks in advance for moving this!

The reason is that SQL operators are used in the `WHERE` clause to perform some type of operation.

Description: 'Searches for matches to words or phrases.'
Subjects:
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'Database'
  - 'Queries'
  - 'PostgreSQL'
  - 'MySQL'
CatalogContent:
  - 'learn-sql'
  - 'paths/analyze-data-with-sql'
---

> CONTAINS() is a predicate used to search for matches to words or phrases in a column.

Reviewer: ```suggestion
In SQL, CONTAINS is a predicate used to search for matches to words or phrases in a column.
```

Reviewer: Can we also mention something about the difference between this full-text search and the [LIKE](https://www.codecademy.com/resources/docs/sql/operators/like) operator?

> ## Code Example

Reviewer: ```suggestion
## Syntax
```

> `CONTAINS()` takes two arguements. A column, and the word/phrase to search for.

Reviewer: ```suggestion
`CONTAINS()` takes two arguments. A column, and the word or phrase to search for.
```

Reviewer: Let's move this explanation underneath the syntax block below! Notice I've suggested a heading change here to "Syntax"

```sql
SELECT column_name
FROM table_name
WHERE CONTAINS(column_name, 'word');
```

> 

Reviewer: ```suggestion

## Example

```

```sql

> SELECT names

Reviewer: ```suggestion
SELECT *
```

Reviewer: Thinking we can select all the info where a person's first name is a specific value

FROM contacts

> WHERE CONTAINS(names, 'Paul');

Reviewer: ```suggestion
WHERE CONTAINS(first_name, 'Paul');
```

Reviewer: I changed this to target the `first_name` column as it's more specific than `names` in my opinion. We can discuss if you want to try something else as well

```

> In this example, we will search the `names` column for any matches to the name `Paul`.

Reviewer: ```suggestion
In this example, we will search the `first_name` column for any matches to the name `Paul`.
```

Reviewer: I would also add one more sentence explaining what we get back from the search!
